llms/mlx_lm/utils.py

```diff
@@ -123,7 +156,8 @@ def generate(
     model: nn.Module,
     tokenizer: PreTrainedTokenizer,
     prompt: str,
-    temp: float = 0.0,
+    temp: float,
```

Comment:
Keep the `temp` and `repetition_penalty` optional to maintain backward compatibility.